app/src/main/java/com/kabouzeid/gramophone/views/AutoTruncateTextView.java

```diff
@@ -115,24 +115,24 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
      * @param truncatedText The string after truncation
      */
     public void initiateTruncateText(final String originalText, final String truncatedText) {
+        if (!originalText.endsWith(TRUNCATED_MARKER)) {
+            text = originalText;
+        }
+        this.truncatedText = truncatedText;
+
+        final TouchInterceptHorizontalScrollView scrollView = getTouchInterceptHorizontalScrollView();
```

Comment:
As long as the scrollview never shows up as null, which I don't think it will, this will do just fine. Good work!